### DISCUSSION TITLE
Automating Chrome Launch and Login with Browserless in Docker

### DISCUSSION THREAD

**Discussion by emirhanerturkx** [browserless/chrome#4456] [General]
Hello, I am using Browserless with Docker, but when I access the opened port, a code editor screen appears, and I also need to press the start button for Chrome to launch. How can I configure it so that Chrome automatically launches on the opened port and logs into the desired address without needing these steps?

**Comment by joelgriffith** [comment]
I'm not sure I follow your request -- are you wanting to use the code editor to write an automated script? That editor is there for debugging purposes but doesn't allow running code on a schedule or interval. This is mostly left up to you on how you want to handle those aspects?
